### PR TITLE
chore: deployment pipeline details

### PR DESCRIPTION
**Pull request by ReubenRobertsHEE**
For the FE n00bs :)

NO-TICKET

### CODE REVIEW

**Comment by john12321** [comment]
Really needed to emphasise pt 2 in that last commit !

**Review by DorisWongNHS — APPROVED** [review]

**Review by Judge40 — DISMISSED** [review]
I'm happy as is, but I'm wondering whether it might make sense to have a sign in utility function that wrapped the wait, visit and sign in steps. We'd only have the explanation in a single place then and would have some more flexibility if we ever needed to change from a 30 second token refresh.

**Comment by john12321** [comment]
> I'm happy as is, but I'm wondering whether it might make sense to have a sign in utility function that wrapped the wait, visit and sign in steps. We'd only have the explanation in a single place then and would have some more flexibility if we ever needed to change from a 30 second token refresh.

Should have stayed away from this 'NO-TICKET' 🤣

File: models/ProfileToFormRPartAInitialValues.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export function ProfileToFormRPartAInitialValues(
     completionDate: programme?.programmeCompletionDate ?? null,
     trainingGrade: "",
     startDate: programme?.startDate ?? null,
-    programmeMembershipType: programme?.programmeMembershipType ?? "",
+    programmeMembershipType: "",
```
Comment (ReubenRobertsHEE) [inline models/ProfileToFormRPartAInitialValues.ts:56]: Perhaps this should really go into another PR - I noticed that the dropdown for new forms was being prepopulated with 'MILITARY'. On staging for some reason the dropdown appears blank which is why I missed it (though it does actually have the value if you proceed to review and submit the form!)

File: cypress/component/forms/formr-part-a/FormA.cy.tsx [hunk 2]
```diff
@@ -336,6 +336,14 @@ describe("FormA (form creation)", () => {
       .find(".react-select__option")
       .first()
       .click();
+    cy.get(
```
Comment (ReubenRobertsHEE) [inline cypress/component/forms/formr-part-a/FormA.cy.tsx:339]: Tweak to component test to match change to ProgrammeMembershipType default

**Review by Judge40 — APPROVED** [review]
